frontend/public/locales/en/connection.json

```diff
@@ -15,10 +16,10 @@
 
     "connections.actions.enableClusterDiscovery": "Enable cluster discovery",
 
-    "modal.delete.title.single": "Delete provider connection",
-    "modal.delete.content.single": "Deleting <bold>{{name}}</bold> is irreversible. The provider connection will no longer be available for creating new clusters, but clusters that were previously created using the connection are not affected. Are you sure that you want to continue?",
-    "modal.delete.title.batch": "Delete {{number}} provider connections",
-    "modal.delete.content.batch": "Deleting a provider connection is irreversible. The provider connection will no longer be available for creating new clusters, but clusters that were previously created using the connection are not affected.",
+    "modal.delete.title.single": "Delete credentials",
```

Comment:
We should have an associated PR for DOC changes needed and keep a running list of changes that need to be made as it will be easier to keep track of as we do it than to come back later and catch everything.  Is there a DOC issue?

Reply:
There is a doc issue but I have already reached out to the doc team